README.md

```diff
@@ -31,84 +31,84 @@ userPassword: readonly
+The main directory data comes from the `main.d` folder. All `.ldif` files in there are loaded in lexical order and parsed as templates to create the data provided by the idmd LDAP endpoint.
 
-Whenever any of the ldif files are changed, added or removed, make sure to reload kidmd with `systemctl reload kopano-kidmd`.
+Whenever any of the ldif files are changed, added or removed, make sure to reload idmd with `systemctl reload libregraph-idmd`.
 
-Kidmd listens on `127.0.0.1:10389` by default and does not ship with any default users. Example configuration is installed (commented) and also available in the `/usr/share/doc/kopano-kidmd/examples` directory.
+idmd listens on `127.0.0.1:10389` by default and does not ship with any default users. Example configuration is installed (commented) and also available in the `/usr/share/doc/libregraph-idmd/examples` directory.
 
 ##### Add new users using the `gen newusers` command
 
-Kidmd provides a way to create ldif data for new users using batch mode similar to the unix `newusers` command using the following standard password file format:
+idmd provides a way to create ldif data for new users using batch mode similar to the unix `newusers` command using the following standard password file format:
 
 ```bash
-uid:userPassword:uidNumber:gidNumber:cn,[mail][,kopanoAliases...]:ignored:ignored
+uid:userPassword:uidNumber:gidNumber:cn,[mail][,libregraphAliases...]:ignored:ignored
 ```
 
 For example, like this:
 
 ```bash
-cat << EOF | kopano-kidmd gen newusers - --min-password-strength=4 > /etc/kopano/kidm/ldif/main.d/50-users.ldif
-jonas:passwordOfJonas123:::Jonas Brekke,jonas@kopano.local::
+cat << EOF | libregraph-idmd gen newusers - --min-password-strength=4 > /etc/libregraph/kidm/ldif/main.d/50-users.ldif
+jonas:passwordOfJonas123:::Jonas Brekke,jonas@lg.local::
 timmothy:passwordOfTimmothy456:::Timmothy Schöwalter::
 EOF
 ```
 
-This outputs an LDIF template file which you can modify as needed. When done run `systemctl reload kopano-kidmd` to make the new users available. Keep in mind that some of the attributes must be unique.
+This outputs an LDIF template file which you can modify as needed. When done run `systemctl reload libregraph-idmd` to make the new users available. Keep in mind that some of the attributes must be unique.
 
 ##### Quick start with demo users
 
-To add the standard Kopano demo users use the following command (you might want to remove existing ldif configuration with `rm /etc/kopano/kidm/ldif/main.d/*.ldif` before generating the demo users to avoid conflicts).
+To add standardize demo users the following command can be used (you might want to remove existing ldif configuration with `rm /etc/libregraph/idm/ldif/main.d/*.ldif` before generating the demo users to avoid conflicts).
 
 ```bash
-/usr/share/kopano-kidmd/generate-demo-users-ldif > /etc/kopano/kidm/ldif/main.d/10-main.ldif
+/usr/share/libregraph-idmd/generate-demo-users-ldif > /etc/libregraph/idm/ldif/main.d/10-main.ldif
```

Comment:
This section is problematic as it describes an installation done by packaging. I suggest to remove it for now - also the generate-demo-users-ldif script has been removed.